packages/ffe-cards-react/src/IconCard/IconCard.md

```diff
@@ -0,0 +1,49 @@
+Ikonkort er kort med en gitt maksbredde og et venstrestilt ikon.
+Den tilbyr alle underkomponenter (CardName, Title, Subtext og Text) og plasserer dem i
+gitt rekkefølge nedover i kortet, med teksten venstrestilt, til høyere for ikonet.
+
+```jsx
+const { GrafOppIkon } = require('../../../ffe-icons-react');
+
+<React.Fragment>
```

Comment:
Tror ikke du trenger å wrappe `IconCard` i `Fragment`?

Reply:
Nei, riktig. Jeg hadde to på et tidspunkt, og glemte å fjerne fragment da jeg fjernet det andre kortet.